rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -944,7 +944,20 @@ void ppu_thread::cpu_task()
 		}
 		case ppu_cmd::initialize:
 		{
-			cmd_pop(), ppu_initialize(), spu_cache::initialize();
+			cmd_pop();
+
+			while (!g_fxo->is_init<rsx::thread>() && !is_stopped())
```

Comment:
RSX thread may be never created in CLI-only elfs. It looks like it's gonna stall forever in this case.

Reply:
Actually it turns out this check has no meaning here, removed.

Reply:
Actually it turns out this check has no meaning here, removed.

Reply:
Still doesn't work and makes little sense. Started some game with frozen window, it also ignored manual startup gui setting.